ui.apps/src/main/content/jcr_root/apps/etoolbox-query-kit/console/query/.content.xml

```diff
@@ -89,13 +89,19 @@
                                 jcr:primaryType="nt:unstructured"
                                 sling:resourceType="granite/ui/components/coral/foundation/collection/actionlink"
                                 granite:id="openSettingsButton'"
-                                action="foundation.dialog"
-                                target=".eqk-ui"
+                                action="foundation.link"
+                                href="/mnt/overlay/commerce/gui/content/v2/catalogs/createcatalogwizard.html${requestPathInfo.suffix}"
```

Comment:
Something's wrong with link, yeah?) Why "create catalog wizard"?

Reply:
sorry, it's a typo)